eng/mgmt/mgmtmetadata/managednetwork.txt

```diff
@@ -4,11 +4,11 @@ Commencing code generation
 Generating CSharp code
```

Comment:
please follow the name pattern for this file, i.e. managednetwork_resource-manager.txt

Reply:
Hi, @erich-wang , to fix it, I modify the read.md in swagger, but the new PR have not been approved yet.  To solve this issue for now, does it work if I manually rename the file?